ui/store/actions.js

```diff
@@ -2738,6 +2723,16 @@ export function estimateGas(params) {
   return promisifiedBackground.estimateGas(params);
 }
 
+export async function updateTokenType(tokenAddress) {
+  let token = {};
+  try {
```

Comment:
Guess you felt loading indicator wasn't required here?

Reply:
its handled at the callsite now, the loading indicator is shown anytime we updateAsset to a token asset.